extensions/wikia/Staging/Staging.setup.php

```diff
@@ -29,5 +29,6 @@
  * @author macbre
  * @see PLATFORM-664
  */
+$wgBaseShareKeyPrefix = $wgSharedKeyPrefix;
 $wgCachePrefix = gethostname() . '-' . wfWikiID(); // e.g. staging-s3-muppet / sandbox-qa02-glee / ...
-$wgSharedKeyPrefix = gethostname() . '-' . $wgSharedKeyPrefix; // e.g. staging-s3-wikicities
+$wgSharedKeyPrefix = Wikia::getCurrentServerSharedKeyPrefix();
```

Comment:
`wgSharedKeyPrefix` used to overwrite itself here using it's original value as part of the key.

This made updating it for different environments difficult so I ended up saving it's original value in `$wgBaseShareKeyPrefix` and using a couple helper methods in `Wikia.php` to construct the key.